powerdns_recursor/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG - powerdns_recursor
 
+## 1.2.0 / 2019-01-04
+
+* [Added] Support Python 3. See [#2789](https://github.com/DataDog/integrations-core/pull/2789).
+* [Added] Command to validate metadata. See [#2269](https://github.com/DataDog/integrations-core/pull/2269).
```

Comment:
Accident?

Reply:
PR 2269 modified the powerdns_recursor metadata.csv file to remove duplicate fields so it'd pass the validation that the PR was introducing, I believe thats why it got added here. I manually removed it from the Changelog